### PR TITLE
Feature/border announcement

### PR DESCRIPTION
**Pull request by PowerSchill**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Adds player announcement regarding the change in colony size when a build is completed.
- Tells the player how to increase their colony size when they try and place a building outside the colony
-

Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -862,6 +862,7 @@
   "com.minecolonies.coremod.leavingcolony": "Farewell, you're leaving %s's colony.",
   "com.minecolonies.coremod.enteringcolonynotify": "Beware, %s just entered %s.",
   "com.minecolonies.coremod.leavingcolonynotify": "Farewell, %s has just left %s.",
+  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for your colony",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for %s.",
```

Maybe the name of the colony here? It gets confusing if you're an officer in multiple colonies.

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 2]
```diff
@@ -279,7 +279,7 @@
   
   "tile.blockhut.messagenopermission": "You don't have permission to open huts in %s.",
   "tile.blockhut.messagenopermissionplace": "You don't have permission to place huts in %s.",
-  "tile.blockhut.messagetoofarfromtownhall": "You need to be closer to your town hall!",
+  "tile.blockhut.messagetoofarfromtownhall": "You need to be closer to your town hall! Expand your colony by upgrading the townhall, or build/upgrade barracks or guardtowers.",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "tile.blockhut.messagetoofarfromtownhall": "You cannot place a building outside of your colony! Expand your colony by upgrading the Town Hall and/or by building/upgrading Barracks and Guard Towers.",
```

**Review by ravenbuilder934 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/minecolonies/lang/en_us.json (hunk 1 above)
Comment (PowerSchill) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: It should be preceded by the "John Doe has completed the %s" that would identify which colony.

Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json:865]: Then 
```suggestion
  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for your colony.
```

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 3]
```diff
@@ -279,7 +279,7 @@
   
   "tile.blockhut.messagenopermission": "You don't have permission to open huts in %s.",
   "tile.blockhut.messagenopermissionplace": "You don't have permission to place huts in %s.",
-  "tile.blockhut.messagetoofarfromtownhall": "You need to be closer to your town hall!",
+  "tile.blockhut.messagetoofarfromtownhall": "You cannot place a building outside of your colony! Expand your colony by upgrading the Town Hall and/or by building/upgrading Barracks and Guard Towers.",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "tile.blockhut.messagetoofarfromtownhall": "You cannot place a building outside of your colony! Expand your colony by upgrading the Town Hall and/or building and upgrading Guard Towers and Barracks.",
```

File: src/main/resources/assets/minecolonies/lang/en_us.json (hunk 1 above)
Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Then 
```suggestion
  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for your colony.",
```

File: src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java [hunk 4]
```diff
@@ -1,9 +1,11 @@
 package com.minecolonies.coremod.util;
 
+import com.ldtteam.structurize.util.LanguageHandler;
 import com.minecolonies.api.colony.IChunkmanagerCapability;
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java:4]: can you undo the reformat too?

Comment (PowerSchill) [reply src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java:4]: Yeah, but I thought the goal was to get all the code to a specific format?

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java:4]: yeah, but it's difficult to see changes in a PR with that

File: src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java [hunk 5]
```diff
@@ -334,11 +339,21 @@ public static void claimChunksInRange(
                 {
                     continue;
                 }
+                else
+                {
+                    areAllChunksAdded = false;
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java:344]: That var seems a bit confusing, might need a better name. Maybe "wereChunksAdded" ?

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java:344]: Do we need this var at all?

Comment (PowerSchill) [reply src/main/java/com/minecolonies/coremod/util/ChunkDataHelper.java:344]: Yes because of the nested loop.

**Pull request closed without merging**